apps/src/templates/RotateContainer.jsx

```diff
@@ -62,7 +62,7 @@ export default class RotateContainer extends React.Component {
       // with a white background, so that if you scroll off of the inner container
       // you see white instead of the codeApp
 
-      const scale = screen.height / 900;
+      const scale = screen.height / 1400;
       width = window.screen.width / scale;
       height = window.screen.height;
```

Comment:
Here, we make the assumption that iOS and Android measure width and height the same, but unfortunately, they don't - the width on iOS is the narrow direction and the height is the tall direction. For android, the width and height are dependent on screen orientation. i.e. on my (Android) phone, if I load in landscape mode and then turn the phone to portrait mode, I see this:
<img src="https://user-images.githubusercontent.com/8324574/78197265-aa756680-7439-11ea-8663-163b61773fed.jpg" width="150"/>

To handle this, we can use this logic:
```
      const windowHeight = Math.max(window.screen.width, window.screen.height);
      const windowWidth = Math.min(window.screen.width, window.screen.height);
      const scale = windowHeight / 1400;
      width = windowWidth / scale;
      height = windowHeight;
```